Assets/Scripts/Constants.cs

```diff
@@ -0,0 +1,48 @@
+public static class Constants
+{
+    /// <summary>
+    /// All the untyped event names
+    /// </summary>
+    /// <remarks>
+    /// All these event names were pulled straight from the list generated, so I can not comment on all these
+    /// </remarks>
+    public static class Events
+    {
+        public const string OnPause = "OnPause";
+        public const string OnResume = "OnResume";
+        public const string OnPrimaryFire = "OnPrimaryFire";
+        public const string OnSecondaryFire = "OnSecondaryFire";
+        public const string OnCallScavenger = "OnCallScavenger";
+        public const string OnOrderScavenger = "OnOrderScavenger";
+        public const string OnOrderSubmitScavenger = "OnOrderSubmitScavenger";
+        public const string OnStopScavenger = "OnStopScavenger";
+        public const string OnReload = "OnReload";
+        /// <summary>
+        /// FIXME: This event seems to only be called in their typed counterpart
+        /// </summary>
+        public const string OnDeath = "OnDeath";
+        /// <summary>
+        /// FIXME: This event seems to only be called in their typed counterpart
+        /// </summary>
+        public const string OnRespawn = "OnRespawn";
+    }
+    /// <summary>
+    /// All the typed event names
+    /// </summary>
+    /// <remarks>
+    /// All these event names were pulled straight from the list generated, so I can not comment on all these
+    /// </remarks>
+    public static class TypedEvents
+    {
+        // TODO : Add all the typed event names cause i couldnt be bothered -nemo
+    }
```

Comment:
Probably a better way to handler this indeed. we should migrate toward these in the future

Reply:
I just saw a dozen hard coded EventManager.AddListener("OnPause", ___) and figured a single typo somewhere could create a hard to find bug

Reply:
at first I wanted to use an enum but figured out it would create a lot of conflict as well. I guess const class is better indeed

Reply:
the structure might need a little work though, i didnt want to go crazy on the class nesting (which i use as "folders")